salt/runners/jobs.py

```diff
@@ -31,6 +32,20 @@
 log = logging.getLogger(__name__)
 
 
+def master():
+    '''
+    Return the actively executing runners for the master
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt-run jobs.master
+    '''
+    jobs = salt.utils.master.get_running_jobs(__opts__)
+    return jobs
```

Comment:
Nitpick: collapse into one line. :)

Reply:
meh. but because its mike, i'll do it for you :D

Reply:
VICTORY! 🎉